module-domain/src/main/kotlin/site/yourevents/guest/service/OwnerNicknameService.kt

```diff
@@ -0,0 +1,37 @@
+package site.yourevents.guest.service
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import site.yourevents.guest.domain.Guest
+import site.yourevents.guest.port.`in`.OwnerNicknameUseCase
+import site.yourevents.guest.port.out.OwnerNicknamePersistencePort
+import site.yourevents.invitation.exception.InvitationNotFoundException
+import site.yourevents.invitation.port.`in`.InvitationUseCase
+import site.yourevents.member.exception.MemberNotFountException
+import site.yourevents.member.port.`in`.MemberUseCase
+import java.util.*
+
+@Service
+@Transactional
+class OwnerNicknameService(
+    private val ownerNicknamePersistencePort: OwnerNicknamePersistencePort,
+    private val memberUseCase: MemberUseCase,
+    private val invitationUseCase: InvitationUseCase
+) : OwnerNicknameUseCase {
+    override fun createOwnerNickname(memberId: UUID, invitationId: UUID, nickname: String): Guest {
```

Comment:
해당 메서드는 ownerNickname을 생성한다기보다, 참석 정보를 가지는 객체를 만드는 메서드이기에 해당 메서드가 가지고 있는 책임을 명확히 드러내야한다고 생각합니다. 따라서 다음과 같이 다른 로직에서도 재사용이 가능하도록 메서드명을 수정하는 것이 좋아보입니다..!!

그리고 컨벤션에 따라 매개변수가 3개 이상이므로, 줄바꿈 부탁드립니다!
```suggestion
    override fun createGuest(
        memberId: UUID, 
        invitationId: UUID, 
        nickname: String
    ): Guest {
```

Reply:
매우 동의하는 부분입니다. 정완님 리뷰를 보고 네이밍이 잘못되었다는것을 확실히 느낍니다!! 감사합니다!! 수정하였습니다.